<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Маршруты</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="filterDataButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Цена по возрастанию</string>
      </property>
     </widget>
     <widget class="QPushButton" name="filterData2Button">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Цена по убыванию</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRem">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить запись</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditID">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить запись</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPrice">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRoute">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditClient">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>0</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Route</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>80</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>160</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="filterDataButton_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>330</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Имена a-z</string>
      </property>
     </widget>
     <widget class="QPushButton" name="filterDataButton_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>330</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Имена z-a</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Клиенты</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemPerson">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>160</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить Запись</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIDPerson">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditClient_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdd_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>50</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAge">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="filterDataButton_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Имена a-z</string>
      </property>
     </widget>
     <widget class="QPushButton" name="filterDataButton_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Имена z-a</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
